src/utils.py

```diff
@@ -69,3 +72,32 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def get_semver_user_input(body):
```

Comment:
we need to check that OWNER comment here.

Reply:
this call is nested underneath the approvers check in `labels_by_user_input.py`

Reply:
ACK, missed it :)